docs/ext.py

```diff
@@ -0,0 +1,94 @@
+from docutils.parsers.rst import directives, roles, nodes
```

Comment:
We should name this file something like `sphinx_pdk_roles.py` -- see https://github.com/google/gf180mcu-pdk/pull/39/files#r938371142

Reply:
Done here and I will change the name in `gf180mcu-pdk` repository also.